virtio-devices/src/vsock/csm/txbuf.rs

```diff
@@ -62,11 +62,13 @@ impl TxBuf {
         // it would later be discarded (when `self.data.is_some()`). Apparently, clippy fails
         // to see this, and insists on issuing some warning.
         #[allow(clippy::redundant_closure)]
-        let data = self.data.get_or_insert_with(||
-                // Using uninitialized memory here is quite safe, since we never read from any
-                // area of the buffer before writing to it. First we push, then we flush only
-                // what had been previously pushed.
-                Box::new(unsafe {mem::MaybeUninit::<[u8; Self::SIZE]>::uninit().assume_init()}));
+        let data = self.data.get_or_insert_with(|| {
+            // Using uninitialized memory here is quite safe, since we never read from any
+            // area of the buffer before writing to it. First we push, then we flush only
+            // what had been previously pushed.
+            #[allow(clippy::uninit_assumed_init)]
+            Box::new(unsafe { mem::MaybeUninit::<[u8; Self::SIZE]>::uninit().assume_init() })
+        });
```

Comment:
This has been solved from the virtio-vsock maintainers on the Firecracker side: https://github.com/firecracker-microvm/firecracker/blob/main/src/devices/src/virtio/vsock/csm/txbuf.rs#L52-L54

Reply:
I fixed it. The PR is ready now.